cinema/views.py

```diff
@@ -0,0 +1,42 @@
+from django.http import JsonResponse, HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.shortcuts import get_object_or_404
+from rest_framework.parsers import JSONParser
+from .models import Movie
+from .serializers import MovieSerializer
+
+@csrf_exempt
```

Comment:
Using `@csrf_exempt` is not recommended as it disables CSRF protection, which is a security risk. Consider using Django REST Framework's `@api_view` decorator, which handles CSRF protection appropriately.